api/src/services/email.service.ts

```diff
@@ -528,7 +528,8 @@ export class EmailService {
         label: this.polyglot.t('rentalOpportunity.applicationsDue'),
         value: dayjs(listing.applicationDueDate)
           .tz(process.env.TIME_ZONE)
-          .format('MMMM D, YYYY'),
+          .format('MMMM D, YYYY [at] h:mma PST'),
```

Comment:
Issue: We shouldn't be hardcoding the PST. Instead it should be `z`. That is because in the summer it is daylight saving time so instead of PST it would be PDT